### PR TITLE
EDU-471 cloud docs refresh branch

### PR DESCRIPTION
**Pull request by flossypurse**
**PREVIEW**: https://temporal-documentation-git-edu-471-q3-cloud-docs.preview.thundergun.io/cloud/

This PR does this following:
- Reorganizes Pricing and Support content into `/cloud/introduction`
  - New Operating envelope content added there as well. Contains commits from https://github.com/temporalio/documentation/pull/2154
- Reorganizes Namespaces and Certificate content into `/cloud/account-setup`
  - Reorged a new User guide into this category as well.
- Migrated How to set up Prom and Grafana with Cloud KB article into Cloud metrics guide
  - Adds metrics descriptions and example usage: https://github.com/temporalio/documentation/pull/2222
- Migrated Namespace naming considerations into Namespace guide
- Renamed guides to match the rest of the docs naming structure
- Added redirects for movement and renaming
- Added a bunch of tags to cloud `docs-src` nodes

### CODE REVIEW

File: docs-src/cloud/support-intro.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+title: How do I get support from Temporal
+sidebar_label: Support
+description: We have a very active community Slack and a very active community forum where you can post questions and ask for help. We also offer ticketing system for Temporal Cloud customers.
+tags:
+  - temporal-cloud
+  - support
+  - introduction
+---
+
+There is an active [community Slack](https://temporalio.slack.com) and an active [community Discourse forum](https://community.temporal.io/) where you can post questions and ask for help. We also offer ticketing system for Temporal Cloud customers.
```
Comment (djmagee) [inline docs-src/cloud/support-intro.md:12]:
```suggestion
We have an active [community Slack](https://temporalio.slack.com) and an active [community Discourse forum](https://community.temporal.io/) where you can post questions and ask for help.
We also offer a ticketing system for Temporal Cloud customers.
```

File: docs-src/cloud/support-intro.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+id: support-intro
+title: How do I get support from Temporal
+sidebar_label: Support
+description: We have a very active community Slack and a very active community forum where you can post questions and ask for help. We also offer ticketing system for Temporal Cloud customers.
```
Comment (djmagee) [inline docs-src/cloud/support-intro.md:5]:
```suggestion
description: We have an active community Slack and an active community forum where you can post questions and ask for help. We also offer a ticketing system for Temporal Cloud customers.
```

File: docs-src/cloud/cloud-intro.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  - [Operational envelope](/cloud/operating-envelope-intro)
+  - [Pricing](/cloud/pricing-intro)
+  - [Support](/cloud/support-intro)
+- [Get started with Temporal Cloud](/cloud/get-started)
+- [Account setup](/cloud/account-setup)
+  - [Manage Namespaces](/cloud/namespaces-intro)
+  - [Manage certificates](/cloud/certificates-intro)
+  - [Manage users](/cloud/users-intro)
+- [Metrics setup](/cloud/metrics-intro)
+- [SAML](/cloud/saml-intro)
```
Comment (djmagee) [inline docs-src/cloud/cloud-intro.md:26]: These two items appear in the other order in the sidebar.

File: docs-src/cloud/operating-envelope-intro.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+---
+id: operating-envelope-intro
+title: What is the operational envelope of Temporal Cloud?
```
Comment (djmagee) [inline docs-src/cloud/operating-envelope-intro.md:3]:
```suggestion
title: What is the operating envelope of Temporal Cloud?
```

File: assembly/guide-configs/cloud/pricing.json [hunk 5]
```diff
@@ -1,17 +1,42 @@
 {
   "file_name": "pricing.md",
   "id": "pricing",
-  "file_dir": "cloud",
-  "title": "Temporal Cloud pricing",
+  "file_dir": "cloud/introduction",
+  "title": "What is the Temporal Cloud pricing",
```
Comment (djmagee) [inline assembly/guide-configs/cloud/pricing.json:5]:
```suggestion
  "title": "What is the pricing for Temporal Cloud?",
```

File: assembly/guide-configs/cloud/support.json [hunk 6]
```diff
@@ -0,0 +1,22 @@
+{
+    "file_name": "support.md",
+    "id": "support",
+    "file_dir": "cloud/introduction",
+    "title": "What kind of support does Temporal Cloud provide",
```
Comment (djmagee) [inline assembly/guide-configs/cloud/support.json:5]:
```suggestion
    "title": "What kind of support does Temporal Cloud provide?",
```

File: changelog/march-release-2023.md [hunk 7]
```diff
@@ -16,4 +16,4 @@ Since our last change log in January 2023, we published the following content:
 
 - The Temporal Cloud [Audit Logging guide](/cloud/how-to-manage-audit-logging) is now available.
 
-- Guidance on how to [submit a support ticket](/cloud/how-to-create-a-ticket-for-temporal-support) for Temporal Cloud users is now available.
+- Guidance on how to [submit a support ticket](/cloud/introduction/support#support-ticket) for Temporal Cloud users is now available.
```
Comment (djmagee) [inline changelog/march-release-2023.md:19]:
```suggestion
- Guidance on how to submit a [support ticket](/cloud/introduction/support#support-ticket) for Temporal Cloud users is now available.
```

File: docs-src/cloud/namespace-setup-considerations.md [hunk 8]
```diff
@@ -29,13 +28,13 @@ Before considering an appropriate Namespace configuration, you should be aware o
...
 - A Namespace is an endpoint.
-  To access a Namespace from a Temporal Client requires mTLS authorization, which requires [CA certificates](cloud/how-to-manage-certificates-in-temporal-cloud#ca-certificates).
+  To access a Namespace from a Temporal Client requires mTLS authorization, which requires [CA certificates](/cloud/account-setup/certificates#ca-certificates).
 - [Workflow Id](/workflows#workflow-id) uniqueness is per Namespace.
 - [Task Queue](/workers#task-queue) names are unique per Namespace.
 - Closed Workflow retention is per Namespace.
 - RBAC [permissions](/cloud/#namespace-level-permissions) are implemented at the Namespace level.
 
-## General Guidance
+### General Guidance
```
Comment (djmagee) [inline docs-src/cloud/namespace-setup-considerations.md:37]:
```suggestion
### General guidance
```

File: docs-src/cloud/namespaces-create.md [hunk 9]
```diff
@@ -18,7 +20,7 @@ To create a Namespace, a user must have the Developer or Global Admin account-le
 :::tip
 
 By default, each account has a quota of 10 Namespaces.
-If you want to increase this limit, open a [support ticket](https://docs.temporal.io/cloud/how-to-create-a-ticket-for-temporal-support).
+If you want to increase this limit, open a [support ticket](https://docs.temporal.io/cloud/support-create-ticket).
```
Comment (djmagee) [inline docs-src/cloud/namespaces-create.md:23]:
```suggestion
If you want to increase this limit, open a [support ticket](https://docs.temporal.io/cloud/introduction/support#support-ticket).
```

File: docs-src/cloud/namespaces-create.md [hunk 10]
```diff
@@ -47,7 +49,7 @@ To create a Namespace in Temporal Cloud, gather the following information:
 1. In **Retention Period**, specify a value from 1 to 90 days.
    When choosing this value, consider your needs for Event History versus the cost of maintaining that Event History.
    Typically, a development Namespace has a short retention period and a production Namespace has a longer retention period.
-   (If you need to change this value later, contact [Temporal Support](https://docs.temporal.io/cloud/how-to-create-a-ticket-for-temporal-support).)
+   (If you need to change this value later, contact [Temporal Support](https://docs.temporal.io/cloud/support-create-ticket).)
```
Comment (djmagee) [inline docs-src/cloud/namespaces-create.md:52]:
```suggestion
   (If you need to change this value later, contact [Temporal Support](https://docs.temporal.io/cloud/introduction/support#support-ticket).)
```

File: docs-src/cloud/operating-envelope-throughput.md [hunk 11]
```diff
@@ -0,0 +1,18 @@
...
+  - operations
+---
+
+A Namespace has a default quota of 200 [Actions](/cloud/what-is-an-action) per second with spikes up to 400 Actions per second.
+However, Temporal Cloud can provide more than 150,000 Actions per second.
+
+If your Action rate exceeds your quota, Temporal Cloud throttles Actions until the rate matches your quota.
+Actions like Start or Signal Workflow Execution always receive higher priority than other Actions, even when throttled.
+
+To raise your quota, [create a ticket for Temporal Support](/cloud/support-create-ticket).
```
Comment (djmagee) [inline docs-src/cloud/operating-envelope-throughput.md:18]:
```suggestion
To raise your quota, create a [support ticket](/cloud/introduction/support#support-ticket).
```

File: docs-src/cloud/users-intro.md [hunk 12]
```diff
@@ -0,0 +1,14 @@
...
+description: Learn how to invite users and manage their roles and permissions.
+tags:
+  - temporal-cloud
+  - temporal-cloud-account
+  - introduction
+---
+
+- How to invite users to your Temporal Cloud Account
+- What are the available Account level roles and permissions
+- What are the Namespace level roles and permissions
```
Comment (djmagee) [inline docs-src/cloud/users-intro.md:14]:
```suggestion
- What are the available Account level roles and permissions?
- What are the Namespace level roles and permissions?
```

File: docs-src/cloud/namespaces-create.md (hunk 9 above)
Comment (flossypurse) [reply docs-src/cloud/namespaces-create.md:23]: this should actually be a id and then link magic will catch it.

File: docs-src/cloud/prometheus-grafana-setup-cloud.md [hunk 13]
```diff
@@ -184,21 +189,21 @@ When you run Prometheus locally, set your target address to port 8077 in your Pr
 
 Example:
```
Comment (djmagee) [inline docs-src/cloud/prometheus-grafana-setup-cloud.md:190]: I'm cheating with this suggestion 'cause the issue in question is technically out of scope: 

In the previous line, change `endopint` to `endpoint`.

**Review by djmagee — COMMENTED** [review]
Just a handful of stragglers, most of which I missed in my previous review.

File: assembly/guide-configs/cli/index.json [hunk 14]
```diff
@@ -4,7 +4,7 @@
   "id": "index",
   "title": "Temporal CLI command reference",
   "sidebar_label": "CLI (temporal)",
-  "sidebar_position": 7,
+  "sidebar_position": 9,
```
Comment (lorensr) [inline assembly/guide-configs/cli/index.json:7]: could this go before tctl?

File: docs-src/cloud/emitted-metrics.md [hunk 15]
```diff
@@ -0,0 +1,41 @@
...
+tags:
+  - explanation
+  - observability
+  - temporal-cloud
+  - reference
+---
+
+Temporal tracks the following metrics for your various Namespaces.
+
+- temporal_cloud_v0_frontend_service_error_count - gRPC errors returned aggregated by operation
```
Comment (TimSimmons) [inline docs-src/cloud/emitted-metrics.md:15]: Now that I'm seeing it [live](https://temporal-documentation-git-edu-471-q3-cloud-docs.preview.thundergun.io/cloud/metrics#available-metrics) I'm wondering if we should bold these names 🤔

File: docs/cloud/introduction/index.md [hunk 16]
```diff
@@ -0,0 +1,18 @@
+---
+id: index
+title: Introduction to Temporal Cloud
+description: Learn more about Temporal Cloud's operating envelope, pricing, and support.
+sidebar_label: Introduction
```
Comment (lorensr) [inline docs/cloud/introduction/index.md:5]: the subpages feel a little hidden to me. like i don't know if i would click on introduction to find this info

image

File: docs-src/cloud/operating-envelope-availability.md [hunk 17]
```diff
@@ -0,0 +1,44 @@
...
+title: What is Temporal Cloud's SLA on Availability?
+sidebar_label: Availability
+description: Temporal Cloud provides 99.9% availability of the cloud service and a 99.9% guarantee against service errors.
+tags:
+  - temporal-cloud
+  - operations
+  - explanation
+---
+
+The Temporal Cloud service level agreement (SLA) promises 99.9% availability of the cloud service and a 99.9% guarantee against service errors.
```
Comment (lorensr) [inline docs-src/cloud/operating-envelope-availability.md:12]:
```suggestion
description: Temporal Cloud provides 99.99% availability of the cloud service and a 99.9% guarantee against service errors.
tags:
  - temporal-cloud
  - operations
  - explanation
---

Temporal Cloud provides 99.99% availability, and its service level agreement (SLA) has a 99.9% guarantee against service errors.
```

[website](https://temporal.io/cloud) now says 4 nines availability, but that's not part of the SLA (the SLA is the "guarantee" part on the website)

File: docs-src/cloud/operating-envelope-latency.md [hunk 18]
```diff
@@ -0,0 +1,16 @@
...
+  - temporal-cloud
+  - operations
+  - explanation
+---
+
+Our latency SLO is 200ms per region for p99.
+Latency is greatly influenced by the actual throughput of a single [Workflow Execution](/concepts/what-is-a-workflow-execution).
+Concurrent operations on the same Workflow Execution could lead to increased latency.
+
+Typical latency for starting and signaling Workflow Executions is less than 100ms.
```
Comment (lorensr) [inline docs-src/cloud/operating-envelope-latency.md:16]:
```suggestion
In June 2023, latency measurements over a week period for starting and signaling Workflow Executions were:

- `StartWorkflowExecution`: 90ms p90, 125ms p99
- `SignalWorkflowExecution`: 53ms p90, 95ms p99
- `SignalWithStartWorkflowExecution`: 87ms p90, 116ms p99

We work on improving latency, so these numbers will decrease over time.

Concurrent operations on the same Workflow Execution could lead to increased latency.
```

throughput of WE doesn't make sense to me. if talking about latency of completion of a WE, that's a different topic.

**Review by lorensr — APPROVED** [review]

File: docs-src/cloud/emitted-metrics.md (hunk 15 above)
Comment (flossypurse) [reply docs-src/cloud/emitted-metrics.md:15]: I can take a quick pass at that -

File: docs-src/cloud/pricing-faq.md [hunk 19]
```diff
@@ -0,0 +1,23 @@
...
+sidebar_label: Pricing FAQ
+description: Temporal Cloud pricing FAQ.
+tags:
+  - temporal-cloud
+  - pricing
+  - faq
+  - explanation
+---
+
+Q: What’s the minimum cost to run Temporal Cloud?
```
[the remark this answers is not in the document]
Comment (djmagee) [reply docs-src/cloud/pricing-faq.md:13]: I'm in favor of this idea, which would also make it possible to remove the run-in headings "Q:" and "A:". (I considered also indenting the answers, but I think that the bold questions will provide sufficient visual distinction.)

File: docs-src/cloud/operating-envelope-regions.md [hunk 20]
```diff
@@ -0,0 +1,15 @@
...
+sidebar_label: Regions
+description: Temporal Cloud currently runs in 10 regions in AWS.
+tags:
+  - operations
+  - temporal-cloud
+  - explanation
+---
+
+Temporal Cloud currently runs in 10 regions in Amazon Web Services (AWS).
+Although Temporal Cloud currently runs only on AWS, it works with applications running in any cloud or data center.
```
Comment (lorensr) [inline docs-src/cloud/operating-envelope-regions.md:13]:
```suggestion
Temporal Cloud currently runs in 10 regions in Amazon Web Services (AWS):

- ap-northeast-1
- ap-southeast-1
- ap-southeast-2
- ca-central-1
- eu-central-1
- eu-west-1
- eu-west-2
- us-east-1
- us-east-2
- us-west-2

Although Temporal Cloud currently runs only on AWS, it works with applications running in any cloud or data center.
```

couldn't find this info anywhere on our site or in our docs. this seems like the right place for it.

File: docs/cloud/introduction/index.md (hunk 16 above)
Comment (flossypurse) [reply docs/cloud/introduction/index.md:5]: Noted. We still want to try this out.

File: docs-src/cloud/operating-envelope-availability.md [hunk 21]
```diff
@@ -0,0 +1,44 @@
...
+title: What is Temporal Cloud's SLA on Availability?
+sidebar_label: Availability
+description: Temporal Cloud provides 99.9% availability of the cloud service and a 99.9% guarantee against service errors.
+tags:
+  - temporal cloud
+  - operations
+  - explanation
+---
+
+The Temporal Cloud service level agreement (SLA) promises 99.9% availability of the cloud service and a 99.9% guarantee against service errors.
```
Comment (flossypurse) [inline docs-src/cloud/operating-envelope-availability.md:12]:
```suggestion
description: Temporal Cloud provides 99.99% availability of the cloud service and a 99.9% guarantee against service errors.
tags:
  - temporal-cloud
  - operations
  - explanation
---

Temporal Cloud provides 99.99% availability, and its service level agreement (SLA) has a 99.9% guarantee against service errors.
```

Per @lorensr

File: docs-src/cloud/operating-envelope-latency.md [hunk 22]
```diff
@@ -0,0 +1,21 @@
...
+---
+
+Our latency SLO is 200ms per region for p99.
+In June 2023, latency measurements over a week period for starting and signaling Workflow Executions were:
+
+- `StartWorkflowExecution`: 90ms p90, 125ms p99
+- `SignalWorkflowExecution`: 53ms p90, 95ms p99
+- `SignalWithStartWorkflowExecution`: 87ms p90, 116ms p99
+
+We work on improving latency, so these numbers will decrease over time.
```
Comment (aarohib) [inline docs-src/cloud/operating-envelope-latency.md:19]: Alternate phrasing:
`As we continue working on improving latencies, these numbers will decrease over time`

File: assembly/guide-configs/cloud/manage-users.json [hunk 23]
```diff
@@ -0,0 +1,30 @@
+{
+    "file_name": "users.md",
+    "id": "users",
+    "file_dir": "cloud/account-setup",
+    "title": "How to manage users in Temporal Cloud",
+    "sidebar_label": "Users",
+    "sidebar_position": 3,
+    "description": "Create Namespaces, use Namespace endpoints for access, and obtain Namespace information.",
```
[the remark this answers is not in the document]
Comment (DJSanti) [reply assembly/guide-configs/cloud/manage-users.json:8]: You probably didn't mean to, but the original description was a direct copy of the one from namespaces.md

File: docs-src/cloud/pricing-support.md [hunk 24]
```diff
@@ -0,0 +1,27 @@
+---
+id: pricing-support
+title: What kind of support do I get with Temporal Cloud?
+sidebar_label: Support
+description: With a subscription to Temporal Cloud, you also gain access to our support organization of developer success engineers and solution architects.
```
Comment (DJSanti) [inline docs-src/cloud/pricing-support.md:5]:
```suggestion
description: A subscription to Temporal Cloud grants access to our support team of developer success engineers and solution architects.
```

File: docs-src/cloud/pricing-support.md [hunk 25]
```diff
@@ -0,0 +1,27 @@
...
+tags:
+  - temporal cloud
+  - pricing
+  - support
+  - explanation
+---
+
+With a subscription to Temporal Cloud, you also gain access to our support organization of developer success engineers and solution architects. Our experts assist with a range of work streams from Workflow design reviews to setting up observability to break/fix support with an agreed-upon set of SLAs.
+
+The members of the Temporal Developer Success team are engineers who are Temporal experts and open source contributors. They're backed by the main contributors to the open source Temporal project. They provide deep knowledge of how Temporal works and how you can optimize your deployment.
```
Comment (DJSanti) [inline docs-src/cloud/pricing-support.md:15]:
```suggestion
The Temporal Developer Success team can provide deep knowledge of how Temporal works and how you can optimize your deployment. This team is comprised of engineers who know Temporal inside and out, and continue to contribute to its development.
```

File: docs-src/cloud/configure-metrics-endpoint-using-ui.md [hunk 26]
```diff
@@ -0,0 +1,62 @@
...
+sidebar_label: Configure via UI
+tags:
+  - how-to
+  - observability
+  - temporal cloud
+---
+
+:::note
+
+To view and manage third-party integration settings, your user account must have the Global Admin [Role](https://docs.temporal.io/cloud/#account-level-roles).
```
Comment (flossypurse) [inline docs-src/cloud/configure-metrics-endpoint-using-ui.md:14]:
```suggestion
To view and manage third-party integration settings, your user account must have the Global Admin [Role](/cloud/what-are-the-account-level-roles-for-users-in-temporal-cloud).
```

File: docs-src/cloud/how-to-use-metrics.md [hunk 27]
```diff
@@ -0,0 +1,26 @@
...
+sidebar_label: How to use
+tags:
+  - temporal cloud
+  - promentheus
+  - metrics
+---
+
+Most Temporal Cloud metrics are suffixed with `_count`. This indicates that they behave largely like a [Prometheus counter](https://prometheus.io/docs/concepts/metric_types/#counter). You'll want to use a function like `rate` or `increase` to calculate a per-second rate of increase, or an extrapolated total increase over a time period.
+
+```
```
[the remark this answers is not in the document]
Comment (flossypurse) [reply docs-src/cloud/how-to-use-metrics.md:14]: @TimSimmons can you confirm what syntax this is?
Is it promql?

File: docs-src/cloud/operating-envelope-latency.md (hunk 22 above)
Comment (flossypurse) [inline docs-src/cloud/operating-envelope-latency.md:19]:
```suggestion
As we continue working on improving latencies, these numbers will decrease over time
```

File: docs-src/cloud/operating-envelope-regions.md [hunk 28]
```diff
@@ -0,0 +1,29 @@
...
+| ap-southeast-2 | Asia Pacific (Sydney)    |
+| ca-central-1   | Canada (Central)         |
+| eu-central-1   | EU (Frankfurt)           |
+| eu-west-1      | EU (Ireland)             |
+| eu-west-2      | EU (London)              |
+| us-east-1      | US East (N. Virginia)    |
+| us-east-2      | US East (Ohio)           |
+| us-west-2      | US West (Oregon)         |
+
+Although Temporal Cloud currently runs only on AWS, it works with applications running in any cloud or data center.
```
Comment (flossypurse) [inline docs-src/cloud/operating-envelope-regions.md:27]:
```suggestion
Temporal Cloud currently runs in 10 regions in Amazon Web Services and it works with applications running in any cloud or data center.
```

File: docs-src/cloud/pricing-faq.md [hunk 29]
```diff
@@ -0,0 +1,23 @@
...
+
+**What’s the minimum cost to run Temporal Cloud?**
+
+The Temporal Cloud service is consumption based. You pay only for what you need with no minimum. Basic support has a minimum monthly fee of $200 per month.
+
+**How do I pay for Temporal Cloud?**
+
+Temporal sends a monthly bill based on your consumption. You can pay this bill with a credit card, ACH, wire transfer, or Temporal Credits.
+
+\*_Can I purchase Temporal Cloud through my Amazon, Azure, or Google Cloud Platform marketplace?_
```
Comment (flossypurse) [inline docs-src/cloud/pricing-faq.md:21]:
```suggestion
**Can I purchase Temporal Cloud through my Amazon, Azure, or Google Cloud Platform marketplace?**
```

File: docs-src/cloud/pricing-sso-and-saml.md [hunk 30]
```diff
@@ -0,0 +1,19 @@
...
+  - explanation
+---
+
+We offer single sign-on (SSO) integration using SAML at a monthly fixed fee based on the number of users registered in Temporal Cloud:
+
+| **Users** | **Cost per month** |
+| --------- | ------------------ |
+| 0 to 25   | $200               |
+| 26 to 50  | $300               |
+| 51+       | $500               |
```
[the remark this answers is not in the document]
Comment (flossypurse) [reply docs-src/cloud/pricing-sso-and-saml.md:19]: Provide this feedback to Jim -

File: docs-src/cloud/pricing-support.md [hunk 31]
```diff
@@ -0,0 +1,27 @@
...
+sidebar_label: Support
+description: With a subscription to Temporal Cloud, you also gain access to our support organization of developer success engineers and solution architects.
+tags:
+  - temporal cloud
+  - pricing
+  - support
+  - explanation
+---
+
+With a subscription to Temporal Cloud, you also gain access to our support organization of developer success engineers and solution architects. Our experts assist with a range of work streams from Workflow design reviews to setting up observability to break/fix support with an agreed-upon set of SLAs.
```
[the remark this answers is not in the document]
Comment (flossypurse) [reply docs-src/cloud/pricing-support.md:13]: it is also apparently a commonly used term for this type of support: https://en.wikipedia.org/wiki/Break/fix

File: docs-src/cloud/what-is-an-action.md [hunk 32]
```diff
@@ -5,12 +5,33 @@ sidebar_label: Action
...
   - term
+  - pricing
+  - temporal cloud
   - explanation
 ---
 
-An Action is the fundamental pricing unit in [Temporal Cloud](/concepts/what-is-temporal-cloud).
+Actions are the fundamental consumption pricing unit in Temporal Cloud.
+An Action in Temporal occurs as part of an execution of your Workflow.
+Each time you execute a Temporal Workflow (a Workflow Execution), the associated Actions are collected and ultimately represent the state and progress of your Temporal Application.
```
[the remark this answers is not in the document]
Comment (flossypurse) [reply docs-src/cloud/what-is-an-action.md:15]: I am cautious to change the meaning here. Even if it is wordy.

File: docs-src/cloud/prometheus-grafana-setup-cloud.md [hunk 33]
```diff
@@ -247,7 +252,7 @@ Verify your Prometheus configuration and restart Prometheus.
 If you’re running Grafana as a container, you can set your SDK metrics Prometheus data source in your Grafana configuration.
 See the example Grafana configuration described in the [Prometheus and Grafana setup for open-source Temporal Cluster](/kb/prometheus-grafana-setup) KB article.
 
-## Grafana dashboards setup
+### Grafana dashboards setup
```
Comment (aarohib) [inline docs-src/cloud/prometheus-grafana-setup-cloud.md:255]: Correct this: "In this article, we will configure our dashboards using the UI." on line 259 to "In this section, we will configure our dashboards using the UI."

**Review by aarohib — APPROVED** [review]

**Review by DJSanti — APPROVED** [review]

**Pull request merged**